src/mlpack/methods/random_forest/random_forest_impl.hpp

```diff
@@ -436,38 +456,46 @@ double RandomForest<
   #pragma omp parallel for reduction( + : avgGain)
   for (omp_size_t i = 0; i < numTrees; ++i)
   {
+    Timer::Start("bootstrap");
```

Comment:
I just added these for my own debugging, but I decided to leave them in because they could be interesting.  If you run the binding with `--verbose`, it'll print all the timers collected during the run, which can be nice to figure out exactly where the time is going.  It may be more useful for developers than users, but I've always thought it's reasonably interesting to look at for users too. :)

Reply:
sounds good!